go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/99designs/gqlgen v0.17.36
 	github.com/DataDog/appsec-internal-go v1.4.0
 	github.com/DataDog/datadog-agent/pkg/obfuscate v0.48.0
-	github.com/DataDog/datadog-agent/pkg/remoteconfig/state v0.48.1
+	github.com/DataDog/datadog-agent/pkg/remoteconfig/state v0.52.0-devel.0.20240116182234-ac770f9620c8
```

Comment:
@grantseltzer When is it planned to release this code as part of an agent release? I see it's some kind of dev version.

Reply:
I agree we should wait for an actual release, i'll talk to the relevant team today to figure out when that's planned for.

Reply:
:+1: We're not in a rush with this PR, we can wait for them to release

Reply:
Based on the schedule previous releases it looks like it will be at least a few weeks before the actual release is cut (it's on rc-1 right now). I've changed the code to use the actual string instead of the constant for now, I can go in and use the constant later on.

Reply:
Since we removed the imported constant we should also remove the dependency here